logstash-core/lib/logstash/webserver.rb

```diff
@@ -3,9 +3,40 @@
 require "puma"
 require "puma/server"
 require "concurrent"
+require "thread"
 
 module LogStash
   class WebServer
+    class NullLogger
```

Comment:
since both of these classes are meant as work arounds for behaviour of Puma, have you thought about namespacing them inside the Puma module, instead of Logstash?